src/math/calculation.js

```diff
@@ -187,6 +187,50 @@ p5.prototype.dist = function() {
   }
 };
 
+/**
+ * Calculates the distance between two points and squares this value.
+ *
+ * @method distSq
+ * @param  {Number} x1 x-coordinate of the first point
+ * @param  {Number} y1 y-coordinate of the first point
+ * @param  {Number} x2 x-coordinate of the second point
+ * @param  {Number} y2 y-coordinate of the second point
+ * @return {Number}    distance between the two points, squared
+ *
+ * @example
+ * <code><div>
+//TODO: Example
+ * </code></div>
+ *
+
+ */
+/**
+ * @method distSq
+ * @param  {Number} x1
+ * @param  {Number} y1
+ * @param  {Number} z1 z-coordinate of the first point
+ * @param  {Number} x2
+ * @param  {Number} y2
+ * @param  {Number} z2 z-coordinate of the second point
+ * @return {Number}    distance between the two points, squared
+ */
+p5.prototype.distSq = function() {
+  p5._validateParameters('distSq', arguments);
+  if (arguments.length === 4) {
+    //2D
+    return (
+      Math.pow(arguments[2] - arguments[0], 2) +
```

Comment:
i would recommend against using `Math.pow()` if at all possible. it is extremely expensive, especially compared to multiplying a number by itself.

Reply:
Ill change it to just the numbers multiplying themselves.